src/builddataset.py

```diff
@@ -13,45 +13,38 @@ def ingest(self, filepath):
             print("File path {} does not exist. Exiting...".format(filepath))
             sys.exit()
 
-
         analyzer = Sentiment("apikey.json")
-        inputFile = open(filepath, "r")
-        while True:
-            info = inputFile.readline().strip("\n")
-
-            # if line is empty, end of file is reached
-            if not info:
-                break
 
-            tweeterInfo = info.split(",")
-            tweeterHandle, tweeterLabel = tweeterInfo[0], tweeterInfo[1]
+        with open(filepath, "r") as f:
+            for info in f:
+                info = info.strip()
+                data = info.split(",")
 
+                try:
+                    tweet_index = int(data[0])
+                    if tweet_index not in self.profiles:
+                        self.profiles[tweet_index] = (tweeterLabel, {})
```

Comment:
where is this tweeterLabel come from?

Reply:
The except clause

Reply:
owow im too used to go, forgot that python doesn't care about scoping declarations all that much